src/index.html

```diff
@@ -13,10 +14,53 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;
+    0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="./images/logo.png">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav>
+        <a
+          href="#"
+          class="is-active"
+        >
+          Apple
+        </a>
+        <a href="#">Samsung</a>
+        <a href="#">Smartphones</a>
+        <a
+          class="nav-link"
+          href="#"
+        >
+          Laptops & Computers
+        </a>
+        <a href="#">Gadgets</a>
+        <a href="#">Tablets</a>
+        <a href="#">Photo</a>
+        <a href="#">Video</a>
```

Comment:
Use semantic tags where possible. The navigation links should be enclosed within a `<ul>` with `<li>` items, using the `<nav>` element to define the navigation section.